packages/aws_signature_v4/pubspec.yaml

```diff
@@ -1,15 +1,16 @@
 name: aws_signature_v4
 description: Dart implementation of the AWS Signature Version 4 algorithm, for communication with AWS services.
-version: 0.1.0
-homepage: https://github.com/aws-amplify/amplify-flutter/packages/aws_signature_v4
+version: 0.2.0
```

Comment:
Do you know how this will impact publishing from stable? Should we align the version in next/main/stable for the non-amplify packages?